modules/apps/transfer/types/trace.go

```diff
@@ -84,10 +84,10 @@ func (dt DenomTrace) IsNativeDenom() bool {
 
 // extractPathAndBaseFromFullDenom returns the trace path and the base denom from
 // the elements that constitute the complete denom.
-func extractPathAndBaseFromFullDenom(fullDenomItems []string) (string, string) {
+func extractPathAndBaseFromFullDenom(fullDenomItems []string) (path, basedenom string) {
```

Comment:
no need for named returns here? I do personally prefer the renaming of `path`, `baseDenom`.

Reply:
ah, this now sets off the confusing-results linter. 

confusing-results: unnamed results of the same type may be confusing, consider using named results (revive)

We can disable that or use the named returns.

Reply:
I think that for now, I will leave this as-is, just lmk how you'd like it.